tests/_modules.js

```diff
@@ -0,0 +1,715 @@
+module.exports =  [{ path: '/Users/samrichard/Development/plugabilly/node_modules/ava',
```

Comment:
Understood.

Do you care about test folder structure right now? Such as putting this in ./tests/fixtures?